index.bs

```diff
@@ -130,6 +130,71 @@ much like a data property as possible. Specific guidance in this regard includes
     true. (This will not always be the case, e.g. if a normalization or type conversion step is
     necessary, but should be held as a goal for normal code paths.)
 
+<h3 id="live-vs-static">Consider whether objects should be live or static</h3>
+
+Objects returned from functions, attribute getters, etc.,
+can either be live or static.
+A <dfn>live object</dfn> is one that continues to reflect
+changes made after it was returned to the caller.
+A <dfn>static object</dfn> is one that reflects
+the state at the time it was returned.
+
+Objects that are the way state is mutated are generally live.
+For example, DOM nodes are returned as live objects,
+since they are the API through which attributes are set and other changes
+to the DOM are made.
+They also reflect changes to the DOM made in other ways
+(such as through user interaction with forms).
+Even though the nodes themselves are live,
```

Comment:
Yeah, I was thinking of having a separate paragraph about how live and static objects can basically point to each other arbitrarily, and live vs. static is a distinction made about a single object rather than a part of the graph, but decided that was too much, and sort of half tacked it on to the end here.  Maybe I should just drop it.  Or maybe expand it?